api/src/organizational-entities/infrastructure/repositories/organization-feature-repository.js

```diff
@@ -3,6 +3,7 @@
  */
 import { knex } from '../../../../db/knex-database-connection.js';
 import * as knexUtils from '../../../../src/shared/infrastructure/utils/knex-utils.js';
+import { ApplicationTransaction } from '../../../prescription/shared/infrastructure/ApplicationTransaction.js';
```

Comment:
**to do :** drop ce commit aussi car utilise uniquement ce qui se trouve dans le commit précédent